### PR TITLE
Discord Community Server

### PR DESCRIPTION
**Pull request by foxmask**
adding a new community entry point for the users looking for live chat ;)

### CODE REVIEW

**Comment by virtadpt** [comment]
"The instant invite is invalid or has expired."

**Comment by tessus** [comment]
When I click on the link, I also get also the above error message.
Do I have to create a user first and must be logged in?

**Comment by foxmask** [comment]
the link https://discordapp.com/invite/d2HMPwE leads you to a page which says "continue to discord" and then ask you to login or create an account

**Comment by tessus** [comment]
@foxmask ok, it seems that my browser was stuck in a weird half-logged-on state for discord. (I tried to use it without registering a user, when I clicked 'Open Discord'  and it always wanted to verify my phone number, so I never did. Even though I was never logged in, I had to log out. Very strange.) Anyway, the link works now. Thanks.

File: README.md [hunk 1]
```diff
@@ -369,6 +369,7 @@ Please see the [donation page](https://joplinapp.org/donate/) for information on
 - For bug reports and feature requests, go to the [GitHub Issue Tracker](https://github.com/laurent22/joplin/issues).
 - The latest news are posted [on the Patreon page](https://www.patreon.com/joplin).
 - You can also follow us on <a rel="me" href="https://mastodon.social/@joplinapp">the Mastodon feed</a> or [the Twitter feed](https://twitter.com/joplinapp).
+- You can join the community in live on [the discord server](https://discordapp.com/invite/d2HMPwE) to get help and exchange on joplin.
```
Comment (tessus) [inline README.md]: After pondering for a while, I think the following would be better in English:

You can join the live community on the [JoplinApp discord server](https://discordapp.com/invite/d2HMPwE) to get help with Joplin or to discuss anything Joplin related.

Comment (foxmask) [reply README.md]: done

Comment (tessus) [reply README.md]: Great, thanks.

**Comment by tessus** [comment]
Thanks for adding that info.

**Pull request merged**